src/Models/Feed/FeedSorterer.cs

```diff
@@ -49,7 +49,7 @@ private void GetLastWeekPosts(ref List<SyndicationItem> list)
             DateTime currentDate = DateTime.Now;
             foreach (var item in list)
             {
-                if (currentDate.AddDays(-_WeekDaysCount) <= item.PublishDate)
+                if (item.PublishDate >= currentDate.AddDays(-_WeekDaysCount) )
```

Comment:
DateTime date1 = new DateTime(2009, 8, 1, 0, 0, 0);
DateTime date2 = new DateTime(2009, 8, 1, 12, 0, 0);
int result = DateTime.Compare(date1, date2);
string relationship;

if (result < 0)
   relationship = "is earlier than";
else if (result == 0)
   relationship = "is the same time as";         
else
   relationship = "is later than";

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.datetime.compare?view=net-5.0